src/Popover/PopoverContent.js

```diff
@@ -1,13 +1,10 @@
 import styled from '@emotion/styled';
-import { transparentize } from 'polished';
 
 import Card from '../Card';
 
 const PopoverContent = styled(Card)`
   background: ${props => props.theme.colors.white};
-  box-shadow: 0 4px 8px
-      ${props => transparentize(0.9, props.theme.colors.black)},
-    0 0 1px ${props => props.theme.colors.grey20};
+  box-shadow: ${props => props.theme.boxShadows.elevation2};
```

Comment:
Styled system uses "shadows" as the key for boxShadows so you should be able to get the elevation at `props.theme.shadows.elevation2`